contracts/src/NativeSeiTokensERC20.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {IBank} from "./precompiles/IBank.sol";
+import {console2} from "forge-std/Test.sol";
+
+contract NativeSeiTokensERC20 is ERC20 {
+
+    address constant BANK_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000001001;
+
+    string public denom;
+    IBank public BankPrecompile;
+
+    constructor(string memory denom_) ERC20("", "") {
+        BankPrecompile = IBank(BANK_PRECOMPILE_ADDRESS);
```

Comment:
just curious: how much space would it take to store `BankPrecompile` as a state? Is it just an address?

Reply:
yes it should just be an address.